packages/admin/src/pages/scheme/[schemeId]/editors.getServerSideProps.tsx

```diff
@@ -0,0 +1,106 @@
+import { GetServerSidePropsContext } from 'next';
+import {
+  getSessionIdFromCookies,
+  getUserTokenFromCookies,
+} from '../../../utils/session';
+import {
+  getGrantScheme,
+  getSchemeEditors,
+  isSchemeOwner,
+} from '../../../services/SchemeService';
+
+export type EditorList = {
+  role: 'EDITOR' | 'OWNER';
+  email: string;
+  id: string;
+};
+
+export type Action = {
+  href: string;
+  label: string;
+  ariaLabel: string;
+};
+
+export type UnformattedEditorRow = {
+  key: string;
+  value: string;
+  action?: string | Action | JSX.Element;
+};
+
+const getEditorsServerSideProps = async ({
```

Comment:
can also just call this getServerSideProps and export it directly rather then `export const getServerSideProps = async (ctx: GetServerSidePropsContext) =>
  runGetEditorsAndHandleError(ctx);`

Reply:
Don't see why this is better? There's a clear distinction between business logic and error handling code which we will lose if we do that.